Description: Fix patch for Qt 5.14.2 incompatibility issue.
 Fix for proper build with Qt 5.14.2 due to incompatibility with 
 dhighdpi.cpp under platformplugin.
Author: Arun Kumar Pariyar <openarungeek@gmail.com>
Last-Update: 2020-06-14

--- deepin-qt5dxcb-plugin-5.0.11.orig/platformplugin/dhighdpi.cpp
+++ deepin-qt5dxcb-plugin-5.0.11/platformplugin/dhighdpi.cpp
@@ -63,7 +63,7 @@ void DHighDpi::init()
     }
 
     VtableHook::overrideVfptrFun(&QXcbScreen::logicalDpi, logicalDpi);
-    active = VtableHook::overrideVfptrFun(&QXcbScreen::pixelDensity, pixelDensity);
+    active = 1;
 }
 
 bool DHighDpi::isActive()
